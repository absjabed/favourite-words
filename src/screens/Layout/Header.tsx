import React from 'react';
import {StyleSheet, Text, View} from 'react-native';
const Header = () => {
  return (
    <View style={styles.main}>
      <View style={styles.container}>
            <Text style={styles.text}>Dictonary Word Finder</Text>
      </View>
    </View>
  );
};

export default Header;

const styles = StyleSheet.create({
  main: {
    width: '100%',
    justifyContent: 'center',
    padding: 10,
    paddingHorizontal: 25,
    borderBottomColor: '#eee',
    borderBottomWidth: 1,
    backgroundColor: '#f2f2f2',
  },
  container: {
    width: '100%',
    height: 35,
    maxHeight: 40,
    flexDirection: 'row',
    justifyContent: 'center',
    alignItems: 'center',
  },

  text: {
    flexDirection: 'row',
    fontSize: 16,
    fontFamily:'Ubuntu-Regular',
    justifyContent: 'center',
  },
});
