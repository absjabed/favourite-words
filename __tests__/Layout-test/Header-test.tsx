import 'react-native';
import React from 'react';
import Header from '../../src/screens/Layout/Header';
import renderer, { create } from 'react-test-renderer';

it('Header component renders correctly', () => {
  renderer.create(<Header />);
});

test('Header snapshot', () => {
  const tree = create(<Header/>);
  expect(tree).toMatchSnapshot();
});