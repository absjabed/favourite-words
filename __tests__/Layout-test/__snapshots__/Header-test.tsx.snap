// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#1f1f1f",
      "borderBottomColor": "#9b9b9b",
      "borderBottomWidth": 0.5,
      "justifyContent": "center",
      "padding": 10,
      "paddingHorizontal": 25,
      "width": "100%",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "flexDirection": "row",
        "height": 35,
        "justifyContent": "center",
        "maxHeight": 40,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#ffffff",
          "flexDirection": "row",
          "fontFamily": "Ubuntu-Regular",
          "fontSize": 16,
          "justifyContent": "center",
        }
      }
    >
      Dictionary Word Finder
    </Text>
  </View>
</View>
`;
